src/main/java/bio/terra/workspace/service/buffer/BufferService.java

```diff
@@ -40,9 +38,6 @@ public BufferService(BufferServiceConfiguration bufferServiceConfiguration) {
 
   private ApiClient getApiClient(String accessToken) {
     ApiClient client = new ApiClient();
-    client.addDefaultHeader(
```

Comment:
I'm proposing we remove these one-off header additions for now. This should come back as part of a more comprehensive tracing & MDC propagation story, which we are poised to tackle with the common-lib tracing / monitoring module.